pkg/trace/traceutil/otel_util.go

```diff
@@ -15,6 +16,7 @@ import (
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	"go.opentelemetry.io/collector/pdata/ptrace"
 	semconv117 "go.opentelemetry.io/collector/semconv/v1.17.0"
+	semconv125 "go.opentelemetry.io/collector/semconv/v1.25.0"
```

Comment:
I'm wondering if we can add a version of semconv that is already in the serverless dependency list?

Reply:
https://github.com/DataDog/datadog-agent/pull/25921